scripts/docs/generate-man.sh

```diff
@@ -1,14 +1,11 @@
 #!/usr/bin/env bash
-#
-# Generate man pages for docker/docker
-#
-
-set -eu
+# Generate man pages for docker/cli
+set -eu -o pipefail
 
 mkdir -p ./man/man1
 
 MD2MAN_REPO=github.com/cpuguy83/go-md2man
-MD2MAN_COMMIT=$(grep -F "$MD2MAN_REPO" vendor.conf | cut -d' ' -f2)
+MD2MAN_COMMIT=v1.0.6
```

Comment:
Should we bump the version in vendor.conf to the same? Was there a specific reason to not keep them in sync?